Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Arguments
-ARG NODE_VERSION=lts-alpine
+ARG NODE_VERSION=18.20.3-alpine3.19
```

Comment:
why not LTS and should we use now that image in all services?

Reply:
I am aligning it with what is in .nvmrc and circleci jobs.